python/ray/air/util/data_batch_conversion.py

```diff
@@ -64,9 +63,7 @@ def _convert_batch_type_to_pandas(
     """
     pd = _lazy_import_pandas()
 
-    if isinstance(data, np.ndarray):
-        data = pd.DataFrame({TENSOR_COLUMN_NAME: _ndarray_to_column(data)})
```

Comment:
Could we keep this code path here? I don't think we can deprecate this quite yet